starter-code/public/scripts/controllers/articleController.js

```diff
@@ -5,7 +5,7 @@ var app = app || {};
   const articleController = {};
 
   // COMMENT: What is this function doing? Where is it called? Does it call any other functions, and if so, in what file(s) do those function(s) live?
-  // (put your response in a comment here)
+  // What function? This comment is below an object within an IFFE. It's called here, which means it's immediatel invoked.
   articleController.index = (ctx) => app.articleView.index(ctx.articles);
```

Comment:
it is referring to  'articleController.index = (ctx) => app.articleView.index(ctx.articles);'